File: 100-Days-Series/Day 67/AlphabetPalindromicPattern.py
# Function to print Palindromic Pattern with Alphabet
def getPrintPalindromic(rows):
    for i in range(1,rows+1,1):
        ch=65
        # Print Spaces
        for j in range(0,rows-i,1):
            print(end="  ")
        # Print Increasing Alphabet
        for j in range(1,i+1,1):
            print(chr(ch),end=" ")
            ch+=1
        for j in range(i-1,0,-1):
            print(chr(ch),end=" ")
            ch+=1
        print()

# Let's Start Here
rows=int(input("Enter the number of rows : "))
getPrintPalindromic(rows)

File: 100-Days-Series/Day 67/AlphabetPalindromicPattern.js

// Arrow Functiont to print Palindromic Pattern with Alphabet
function getPrintPalindromic(rows){
    for(let i=1;i<=rows;i++){
        line="";
        ch=65;
        // Collect Spaces
        for(let j=1;j<=rows-i;j++){
            line+="  ";
        }
        // Collect Increasing Alphabet
        for(let j=1;j<=i;j++){
            line+=String.fromCharCode(ch++) + " ";
        }
        for(let j=i-1;j>=1;j--){
            line+=String.fromCharCode(ch++) + " ";
        }
        console.log(line);
    }
}
//Let's Start here
let rows=5;
getPrintPalindromic(rows);